File: TRNMNT.Core/Model/Round/RoundDetailsModel.cs
namespace TRNMNT.Core.Model.Round
{
    public class RoundDetailsModel
    {
        public string RoundId { get; set; }
        public int AParticipantPenalties { get; set; }
        public int SecondParticipantPenalties { get; set; }
        public int AParticipantAdvantages { get; set; }
        public int SecondParticipantAdvantages { get; set; }
        public int AParticipantPoints { get; set; }
        public int SecondParticipantPoints { get; set; }

        public int Countdown { get; set; }
        public bool IsStarted { get; set; }
        public bool IsPaused { get; set; }
        public bool IsCompleted { get; set; }
    }
}

File: TRNMNT/ClientApp/app/core/model/round-details/round-details.model.ts
import { RoundModel } from '../round.models';

export class RoundDetailsModel {
    roundId: string;
    AParticipantPenalties: number;
    BParticipantPenalties: number;
    AParticipantAdvantages: number;
    BParticipantAdvantages: number;
    AParticipantPoints: number;
    BParticipantPoints: number;

    countdown: number;
    isStarted: boolean;
    isPaused: boolean;
    isCompleted: boolean;

    roundModel: RoundModel;
}